ngx-tools/src/update-control-on-input-changes/update-control-on-input-changes.spec.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  SimpleChanges,
+  SimpleChange,
+} from '@angular/core';
+
+import { updateControlOnInputChanges } from './update-control-on-input-changes';
+import { FormControl } from '@angular/forms';
+
+
+describe(`updateControlOnInputChanges`, () => {
+  const changed = {
+    item1: new SimpleChange(undefined, true, true),
+    item2: new SimpleChange('foo', 'foo', false),
+  } as SimpleChanges;
+  const control: FormControl = new FormControl();
+
+
+  test(`should return false if the changes object or key are missing`, () => {
+    expect(updateControlOnInputChanges(null, 'foo', control)).toEqual(false);
+  });
+
+
+  test(`should return true if the value has changed`, () => {
+    expect(updateControlOnInputChanges(changed, 'item1', control)).toEqual(true);
+    expect(control.value).toEqual(changed.item1.currentValue);
+    expect(updateControlOnInputChanges(changed, 'item2', control)).toEqual(false);
```

Comment:
I would either split this into two `test` statements or generalize the test statement to something like `should return a boolean representing whether the control was updated`.

We ultimately want the test name to clearly indicate what is being tested within.